scripts/apps/desks/views/desk-select.html

```diff
@@ -1,12 +1,12 @@
 <div class="dropdown dropdown--noarrow dropdown--boxed dropdown--full-width" dropdown is-open="isOpen" tabindex="">
     <button class="dropdown__toggle dropdown__toggle--line" dropdown__toggle data-test-id="desk-select--handle">
         <span ng-show="!selectedDesk" translate>Select desk</span>
-        <span ng-show="selectedDesk">{{ defaultDesk && defaultDesk._id === selectedDesk._id ? 'Default' : selectedDesk.name || selectedDesk.label }}</span>
+        <span ng-show="selectedDesk && !selectedDesk.restriction">{{ defaultDesk && defaultDesk._id === selectedDesk._id ? 'Default' : selectedDesk.name || selectedDesk.label }}</span>
```

Comment:
We need to put a restriction on it since when the restriction is turned on for the current desk, it does not appear in the dropdown selections, but we can see it in the default value.

![image (10)](https://user-images.githubusercontent.com/73937490/210787809-af3c8978-14fd-4006-9fe1-21fc75cecca4.png)

![image (11)](https://user-images.githubusercontent.com/73937490/210787757-070059d9-e485-42a9-9bfc-f4aa05468a77.png)

Reply:
I'm not saying not to put the restriction **at all**, just not here. I see the component accepts `allDesks, selectedDesk, defaultDesk` parameters, so you should instead filter out desks from there before passing the parameters. Otherwise the desks that can't be sent to, would not show up in any desk select, not only in "send to" panel.